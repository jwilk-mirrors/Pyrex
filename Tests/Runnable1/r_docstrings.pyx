"This is a module docstring."

def f():
	"This is a function docstring."

class C:
	"This is a class docstring."

cdef class T:
	"This is an extension type docstring."

