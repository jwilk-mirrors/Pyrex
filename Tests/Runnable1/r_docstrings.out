This is a module docstring.
This is a function docstring.
This is a class docstring.
This is an extension type docstring.
